sql_metadb/derived_tables/holding_notes.sql

```diff
@@ -0,0 +1,45 @@
+DROP TABLE IF EXISTS holdings_notes;
+CREATE TABLE holdings_notes AS
+WITH notes AS (
+    SELECT 
+    h.instanceid AS instance_id,
+    h.id AS holding_id,
+    jsonb_extract_path_text(h.jsonb, 'hrid') AS holding_hrid,
+    jsonb_extract_path_text(notes.jsonb, 'note') AS note,
+    jsonb_extract_path_text(notes.jsonb, 'holdingsNoteTypeId')::uuid AS note_type_id,
+    notes.ordinality AS note_ordinality
+    FROM 
+    folio_inventory.holdings_record h
+    CROSS JOIN LATERAL jsonb_array_elements(jsonb_extract_path(h.jsonb, 'notes'))
+    WITH ordinality AS notes (jsonb))
+SELECT    
+    n.instance_id,
+    jsonb_extract_path_text(i.jsonb, 'hrid') AS instance_hrid,   
+    n.holding_id,
+    n.holding_hrid,
+    n.note,
+    hntt.name AS note_type_name, 
+    n.note_type_id,
+    n.note_ordinality,
+    hnt.creation_date AS note_date_created,
+    hnt.created_by AS note_created_by_user_id,
+    jsonb_extract_path_text(hnt.jsonb, 'updatedDate') AS note_date_updated,

```

Comment:
The type is probably something other than `text`, but we don't have any sample data to look at.  So let us leave it like this.